src/app/modules/main-module/components/main/main.component.scss

```diff
@@ -12,7 +12,27 @@
   background-color: rgb(78, 78, 78);
   color: white;
 }
-
+.contenedorTeclado {
+  height: 280px;
+  width: 35em;
```

Comment:
porqué el height no lo usais con medidas relativas y no fijas? para las teclas si lo tenéis así

Reply:
Toda la razón, lo he cambiado por esto 
.contenedorTeclado {
  height: 8%;
  width: 35em;
}